src/pages/Home/components/HeroSection/components/mainText/components/fullMovieLink/index.jsx

```diff
@@ -0,0 +1,33 @@
+import styles from './fullMovieLink.module.scss'
+
+export function FullMovieLink() {
+  return (
+    <div className={styles.linkContainer}>
+      <a className={styles.fullMovieLink}>
+        <span className={styles.linkIconContainer}>
+          <svg
```

Comment:
would be better to move svg icon to the separate file .svg and put it to the assets also you can create icon component